src/logger.cpp

```diff
@@ -46,6 +47,11 @@ Logger::Log& Logger::Log::operator<<(const QStringList& t) {
   return *this;
 }
 
+Logger::Log& Logger::Log::operator<<(const QJsonObject& t) {
+  m_data->m_ts << QJsonDocument(t).toJson(QJsonDocument::Indented) << ' ';
```

Comment:
why the extra space at the end?

Reply:
This just maintains consistency with all the other log `operator<<()` methods, which also include a space at the end.

Personally, I find it ever so slightly obnoxious that the logger does this.